### PR TITLE
Add basic Travis integration and a simple test

### PR DESCRIPTION
**Pull request by zachriggle**

### CODE REVIEW

File: tests/testLoadsWithoutCrashing.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import common
+
+def test_loads_witout_crashing():
```
Comment (CaptureTheBox) [inline tests/testLoadsWithoutCrashing.py]: "without"

Comment (gsingh93) [inline tests/testLoadsWithoutCrashing.py]: "without"

Comment (zachriggle) [inline tests/testLoadsWithoutCrashing.py]: Fix'd

**Comment by zachriggle** [comment]
Superceded by #55

**Pull request closed without merging**